src/optimizer/stats/column_stats.cpp

```diff
@@ -43,6 +43,9 @@ uint64_t ColumnStats::GetCardinality() {
 }
 
 std::vector<double> ColumnStats::GetHistogramBound() {
+  if(!(column_type_ == type::Type::INTEGER || column_type_ == type::Type::DECIMAL)) {
```

Comment:
Thanks for adding this! It should be checking any numeric type not limit to integer or decimal. I can refactor this later.